server/src/main/java/org/elasticsearch/action/termvectors/TermVectorsRequest.java

```diff
@@ -621,6 +626,7 @@ public static void parseRequest(TermVectorsRequest termVectorsRequest, XContentP
                     termVectorsRequest.index = parser.text();
                 } else if (TYPE.match(currentFieldName, parser.getDeprecationHandler())) {
                     termVectorsRequest.type = parser.text();
+                    deprecationLogger.deprecated(RestMultiTermVectorsAction.TYPES_DEPRECATION_MESSAGE);
                 } else if (ID.match(currentFieldName, parser.getDeprecationHandler())) {
```

Comment:
cut-and-paste code with wrong message?

Reply:
I'll update this too, as I think it's clearer to use `RestTermVectorsAction` here. I initially used the multi termvectors message because this method is also used in parsing multi termvector requests.